spec/takeaway_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'takeaway'
+describe Takeaway do
+  subject { Takeaway.new }
+  let(:dish) { double("Chicken") }
+  let(:price) { double("£4") }
+  let(:choice) { double(:choice) } 
+
+  describe "#initialize" do
+    it "menu array" do
+      expect(subject.menu).to be_truthy
+    end
+
+    it "empty orders array" do
+      expect(subject.orders).to eq []
+    end
+ 
+  end
+  
+  describe "#print_menu" do
+    it "displays menu" do
+      list_of_dishes = subject.print_menu
```

Comment:
This test is "circular", so it seems it's not actually testing anything — it would be the equivalent of writing something like:
```ruby
my_number = 10 
expect(my_number).to eq(10)
```

What you're looking for is to compare the result of `subject.print_menu` to an *expected* list of dishes, which it should be equal to:

```ruby
expected_list_of_dishes = { rice: 3, chicken: 2, beans: 3, yam: 2, fish: 4, noodles: 2 }
expect(subject.print_menu).to eq(expected_list_of_dishes)
```